jbake.properties

```diff
@@ -1,3 +1,5 @@
+jbakeVersion=2.6.7
```

Comment:
I decided to extract this from all content, including latest docs, but excluding all non-latest docs.
Not sure if you'd rather hard code the jbakeVersion in all the docs that reference GAV's, because you'd need to replace them in the new non-latest docs during a release...

(On optaplanner.org we avoid this by having our docs in the optaplanner repo itself and have optaplanner.org use antora to generate https://www.optaplanner.org/docs - but that's a lot of work, not something I would recommend at this stage)

Reply:
I'd done something similar but only to the parent index.adoc for the last few releases.

Makes sense though.